backend/web/src/frontend/src/views/header/component/TagEditModal.vue

```diff
@@ -70,11 +69,18 @@ export default {
     };
   },
   computed: {
-    ...mapGetters([CATEGORIES]),
+    ...mapGetters([TOTAL_CATEGORIES]),
   },
   methods: {
-    async changeTag({ added }, id) {
-      await CategoryService.changeTag(id, added.element.id);
+    ...mapActions([EDIT_TAG]),
+    ...mapMutations([SHOW_SNACKBAR]),
+    async changeTag({ added }, tagId) {
+      try {
+        await this[EDIT_TAG](tagId, added.element.id);
+        this[SHOW_SNACKBAR](MESSAGES.TAG.EDIT.SUCCESS);
+      } catch (e) {
+        this[SHOW_SNACKBAR](MESSAGES.TAG.EDIT.FAIL);
```

Comment:
```suggestion
        await this.editTag(tagId, added.element.id);
        this.showSnackbar(MESSAGES.TAG.EDIT.SUCCESS);
      } catch (e) {
        this.showSnackbar(MESSAGES.TAG.EDIT.FAIL);
```
이렇게 해도 정상적으로 동작하는데, 위와 같이 작성하신 특별한 이유가 있나요?